src/contentmanager.cpp

```diff
@@ -189,6 +188,63 @@ QString ContentManager::downloadBook(const QString &id)
     return QString::fromStdString(download->getDid());
 }
 
+void ContentManager::pauseBook(const QString& id)
+{
+    qInfo() << "pause";
+    auto& b = mp_library->getBookById(id);
+    kiwix::Download* download;
+    download = mp_downloader->getDownload(b.getDownloadId());
+    download->pauseDownload();
```

Comment:
`auto download = mp_downloader->getDownload(b.getDownloadId());` ?

The `auto` keyword avoid us the write the exact type. If at a moment we change the signature of `getDowload` to return something else (as a smart pointer), we don't have to change this code.
And I prefer to initialize variable directly in the same line it is declared. It may avoid uninitialized variable in the future (refactoring, wrong copy/paste, ...)